src/open_producten/producttypen/admin/externe_code.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+
+from open_producten.producttypen.models import ExterneCode
+
+
+class ExterneCodeInline(admin.TabularInline):
```

Comment:
I don't see this mentioned in #11, is this for https://github.com/maykinmedia/open-producten/issues/13?

In general I'd prefer to have separate PRs for separate issues, I'll allow it in this case since the features are implemented the same way